apiary.apib

```diff
@@ -555,6 +555,11 @@ Get a list of all users.
         + filter (object, optional)
             + ids: `cb8da52a-ce89-4bf6-8f7e-8ee6cb85e3b5`,`f8a57a6f-dd1e-41a3-b8d3-428663f1d09e` (array[string], optional)
             + term: `John` (string, optional) - Filters on first name, last name and email
+            + status: `active`, `deactivated` (array[string], optional) - Filters on status
+                + (enum[string])
+                    + Members
+                        + active - Filters on active users
+                        + deactivated - Filters on deactivated users
```

Comment:
PP: Feels a bit weird to me that we use the past tense for `deactivated` but the present tense for `active`.

Reply:
I think it makes sense since the user is still "active", the user has been "deactivated".